VEDA/settings/base.py

```diff
@@ -110,6 +110,14 @@
 # Python dotted path to the WSGI application used by Django's runserver.
 WSGI_APPLICATION = 'VEDA.wsgi.application'
 
+# NOTE (CCB): JWT_SECRET_KEY is intentionally not set here to avoid production releases with a public value.
```

Comment:
Please don't use my initials for a comment that I did not explicitly write.

Reply:
Oh, I thought this was some sort of the note that needed to here just like in course-discovery. NVM, I will remove :)